ltc/logs/command_factory/logs_command_factory_test.go

```diff
@@ -37,10 +38,12 @@ var _ = Describe("CommandFactory", func() {
 	Describe("LogsCommand", func() {
 
 		var logsCommand cli.Command
+		var appExaminer *fake_app_examiner.FakeAppExaminer
 
 		BeforeEach(func() {
 			commandFactory := command_factory.NewLogsCommandFactory(terminalUI, fakeTailedLogsOutputter, exitHandler)
-			logsCommand = commandFactory.MakeLogsCommand()
+			appExaminer = &fake_app_examiner.FakeAppExaminer{}
+			logsCommand = commandFactory.MakeLogsCommand(appExaminer)
 		})
 
 		It("tails logs", func() {
```

Comment:
Can you please add the test cases also. scenario of tailing when app does not exist.